addons/l10n_id/models/res_bank.py

```diff
@@ -91,6 +91,8 @@ def _get_qr_vals(self, qr_method, amount, currency, debtor_partner, free_communi
                 "cliTrxAmount": int(amount)
             }
             response = _l10n_id_make_qris_request('show_qris.php', params)
+            if response.get('status') == 'failed':
```

Comment:
what other `status` might we get?

If there is an issue communicating with the QRIS endpoint, we raise a validation error. but with this fix, there is no indication that something went wrong?
Should we not raise an error perhaps? @vin-odoo @Juliette-JVAN

Reply:
I guess we should probably display the error we receive.
@nni-odoo Could you confirm if we can do that with the POS?

Reply:
Indeed we should have checked for the response status first. Status is always either "success" or "failed" (reference: https://qris.online/api-doc/create-invoice.php). I think we can do something like this 
```python3
            if response.get("status") == 'failed':
                msg = response.get("data")
                raise ValidationError(_("%s", msg))
            data = response.get('data')

````
the QRIS error will be propagated to PoS interface as well  like the following 
![image](https://github.com/user-attachments/assets/acd24e65-23de-406f-a65c-5cafef097663)